IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoad/Documentation.md

```diff
@@ -0,0 +1,47 @@
+The _IfcRoad_ schema forms part of the Domain Layer of the IFC Model. It extends the ideas concerning infrastructure outlined in the _IfcSharedInfrastructureElements_ schema. It defines concepts specific to road infrastructure.
+
+The in scope road types and related spaces have been identified as the most common and widespread ones across the world: 
+
+- Controlled access highway;
+- Dual carriageway;
+- Single carriageway;
+- Street;
+- Bicycle path;
+- Footpath.
+  In-scope junction types:
```

Comment:
For the sake of readability, I'd include a blank line before and after this line, as well as de-indent it.

Reply:
resolved by change suggestions